### PR TITLE
[Backport] Backport translations and bugfixes to v5

### PR DESCRIPTION
**Pull request by jameswklassen**

### CODE REVIEW

**Review by marcsoiferman — APPROVED** [review]
Looks like a backport

File: wct.conf.json [hunk 1]
```diff
@@ -12,23 +12,18 @@
...
         },
         {
           "browserName": "safari",
-          "platform": "OS X 10.13",
-          "version": ""
-        },
-        {
-          "browserName": "microsoftedge",
-          "platform": "Windows 10",
-          "version": ""
```
Comment (jadasc) [inline wct.conf.json:31]: Are these changes in `wct.conf.json` necessary for v5?

Comment (jameswklassen) [reply wct.conf.json:31]: I'm also backporting the GitHub actions stuff so that the two CI pipelines are identical, so I think this change makes sense.

Comment (jameswklassen) [reply wct.conf.json:31]: Also, the old CI was never using these browsers, since it was running 'local' tests (i.e. not the defined saucelabs browser)

Comment (jadasc) [reply wct.conf.json:31]: Were the changes to this file originally surfaced by an automatic dependency update to `package-lock.json`? Or was that a different repo/problem?

**Review by jadasc — APPROVED** [review]

File: wct.conf.json (hunk 1 above)
Comment (jameswklassen) [reply wct.conf.json:31]: discussed offline

**Pull request merged**